frontend/src/pages/Auth/AdminForm.tsx

```diff
@@ -0,0 +1,214 @@
+import { useAuthContext } from '@authentication/AuthContext'
+import {
+	AutoJoinEmailsInput,
+	getEmailDomain,
+} from '@components/AutoJoinEmailsInput'
+import { Button } from '@components/Button'
+import {
+	AppLoadingState,
+	useAppLoadingContext,
+} from '@context/AppLoadingContext'
+import { useUpdateAdminAndCreateWorkspaceMutation } from '@graph/hooks'
+import {
+	Box,
+	ButtonLink,
+	Callout,
+	Form,
+	Stack,
+	Text,
+	useFormState,
+} from '@highlight-run/ui'
+import { AuthBody, AuthFooter, AuthHeader } from '@pages/Auth/Layout'
+import { Landing } from '@pages/Landing/Landing'
+import analytics from '@util/analytics'
+import { getAttributionData } from '@util/attribution'
+import { message } from 'antd'
+import React, { useEffect, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import * as styles from './AdminForm.css'
+import * as authRouterStyles from './AuthRouter.css'
+
+const COMMON_EMAIL_PROVIDERS = ['gmail', 'yahoo', 'hotmail', 'fastmail', 'protonmail', 'hey.com']
```

Comment:
Just noticed that our CI fails because this file is not formatted

```suggestion
const COMMON_EMAIL_PROVIDERS = ['gmail', 'yahoo', 'hotmail', 'fastmail', 'protonmail', 'hey.com'] as const
```